src/components/Swap/Summary/index.tsx

```diff
@@ -64,7 +68,12 @@ function ConfirmButton({
       return {
         color: 'accent',
         message: <Trans>Price updated</Trans>,
-        icon: AlertTriangle,
+        icon: () => <AlertTriangle color="accent" />,
```

Comment:
Should we fix the issue where having tooltipContent causes the icon to be the wrong color, rather than having to manually recolor the icon here? It seems like a bug in ActionButton

Reply:
good call, fixed